### PR TITLE
MUMMNG-1304 Adds tooltip to widget selector

### PR DESCRIPTION
**Pull request by vertein**
Gif looks a bit choppy, but it's the same settings as the tooltip on the '+' button.

viewtooltip

### CODE REVIEW

**Comment by apetro** [comment]
:+1:

**Comment by keirserrie** [comment]
Looks like space being added after "Simple" messing up centering

File: angularjs-portal-home/src/main/webapp/partials/home-toggle.html [hunk 1]
```diff
@@ -1,4 +1,23 @@
 <span class="layout-chooser" ng-if="toggle">
-  <li ng-class="{active : modeIs('list')}"><a  href="javascript:;" ng-click="switchMode('list')"><i class="fa fa-th"></i></a></li>
-  <li ng-class="{active : modeIs('widgets')}" ><a href="javascript:;" ng-click="switchMode('widgets')"><i class="fa fa-th-large"></i></a></li>
-</span>
\ No newline at end of file
+  <li ng-class="{active : modeIs('list')}">
+    <a  href="javascript:;" ng-click="switchMode('list')"
```
Comment (apetro) [inline angularjs-portal-home/src/main/webapp/partials/home-toggle.html]: I suspect that we ought to be doing something to this `<a` tag to improve its accessibility characteristics.  As in, a `title` or `ARIA`-something-or-other or a screen-reader-only alternative content for the tag?  Link text only visible to screen readers might be brilliant here, in that if I can't see the `fa-th`, what I really want is the text equivalent.  Maybe that should be happening on the `<i` tag?

**Comment by jhanstra** [comment]
Not sure if we've talked about it before, but I kind of like 'Content View' or 'Widget View' more than 'Detail View'...thoughts?

**Comment by keirserrie** [comment]
I could go for Widget View...others have thoughts?

**Comment by vertein** [comment]
Right about the spacing thing, corrected.  Widget View sounds good to me.

**Comment by apetro** [comment]
I think almost any text trying to articulate a text version of this is better than no text, so :+1: for progress.

I suspect that we ought to be getting to some specific, consistent terms of art, names for the features in MyUW that we need to be able to talk about and that users need to be able to talk with us about.  So, eventually, this is "Smart Widgets" vs "Minimal Widgets", and we're quite specific about what we call these, and whether those are proper nouns or "smart widgets" vs "minimal widgets", etc.  

I think we should take whatever steps towards this we can in this story, and if we don't get clear enough about this, we're going to end up revisiting it as the need becomes clearer when KB etc. documentation catches up to talk more about Beta features.

**Comment by keirserrie** [comment]
What do you guys think, does labeling "smart widgets" and "simple widgets" make sense to a user?

**Comment by timlevett** [comment]
:+1:

**Comment by vertein** [comment]
The decision is to now use 'icons' and 'widgets'.  I will update and merge the pr shortly

**Comment by vertein** [comment]
Changed the wording to 'icons' and 'widget's

**Comment by keirserrie** [comment]
Sorry...can you capitalize Icons and Widgets. Going w/sentence case for most things and want to be consistent.

**Comment by jhanstra** [comment]
@keirserrie, I went in and capitalized it and pushed a commit to master.

**Comment by keirserrie** [comment]
@jhanstra THX

**Pull request merged**